packages/core/src/overlay/OverlayHeader.css

```diff
@@ -0,0 +1,31 @@
+.saltOverlayHeader {
+  padding: var(--salt-spacing-100);
+  width: 100%;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: stretch;
+  gap: var(--salt-spacing-100);
+  box-sizing: border-box;
+}
+
+.saltOverlayHeader-body {
```

Comment:
@joshwooding following up on question from previous PR: *"Is "body" the correct way to refer to this?"*

I think it is, as it's the body of the header (multiple text elements that are wrapped together, discrete from the end adornment). Not a strongly-help opinion, though, so happy to hear counter arguments 😄

Reply:
It needs to be aligned with design @navkaur76

Reply:
In figma, the frame containing the main heading, pre-header and description is called "Block", if that helps? The accent bar sits outside of this frame.

Reply:
Closest block in the system is in stepper input?